### ISSUE TITLE
iOS Safari tap while scrolling fires on wrong target

### ISSUE THREAD

**Issue by raegen** [Polymer/polymer#2716]
Platform: iOS, Safari 
Details: While scrolling through a list of repeated elements, tap event is fired on the wrong target even if scrolling very slowly. It's as if the coordinate Y is not updated at all, not until the scrolling animation stops. It's frozen in place the whole time while scrolling - let's say an element A is in the center of the screen. I start scrolling (momentum let's me tap while scrolling) and tap on element D after a second (while still scrolling), The event is fired on element A. The dislocation is not fixed, the event is fired on the element that was in the place of the latter tap right before the scrolling starts.
The moment scrolling stops, if tapped, the event is fired on the correct target.
@azakus this might be gestures.
update: for now I'm avoiding the issue by preventing the tap from firing at all while scrolling. Which gets  me to this: maybe it is actually a desired behavior? The first touch while scrolling stops the scroll and only another one can fire a tap event, that's the way it works in iOS.

**Comment by azakus** [comment]
What version of iOS? Are you using `--webkit-overflow-scrolling: touch` ?

**Comment by mug3n** [comment]
Tested on iOS 8.4 and iOS 9.1. Yes I am.

**Comment by mug3n** [comment]
Any info? Had time to check it out yet?

**Comment by azakus** [comment]
Please provide a JSBin containing a set of reproducible steps.

**Comment by M165437** [comment]
@mug3n Could you ever figure this out? I'm struggling with the exact problem. Thanks!

**Comment by TimvdLippe** [comment]
Thanks for the issue.  Please see [CONTRIBUTING.md](https://github.com/Polymer/polymer/blob/master/CONTRIBUTING.md) for guidelines on filing issues.  Please fill out as many fields in the issue template as possible, and be sure to include a live reproduction so that the team can debug or understand your issue.  You can start from [this jsBin template](https://github.com/Polymer/polymer/blob/master/CONTRIBUTING.md) as a starting point and link it in the issue template under "Live Demo".

Feel free to file a new issue following these instructions.  Thank you!
